documentation/managing_modules.md

```diff
@@ -96,18 +95,72 @@ bolt module install --force
 Install-BoltModule -Force
 ```
 
+## Specify modules in a Bolt project
```

Comment:
I think this is a little confusing if it comes before the "installing modules" section - unless we're expecting all users to manually specify their modules? Perhaps we can move it below that section?
```suggestion
## Manually specify modules in a Bolt project
```